patches/chrome-browser-importer-external_process_importer_client.h.patch

```diff
@@ -1,22 +1,16 @@
 diff --git a/chrome/browser/importer/external_process_importer_client.h b/chrome/browser/importer/external_process_importer_client.h
-index 9451b0917536e73b42c596527112d3119b2c4cc5..63ca4957684b5c15fa76a6c53197376633e9e096 100644
+index 9451b0917536e73b42c596527112d3119b2c4cc5..839f905b403a8103ecf22aba16dd03f52f7cf796 100644
 --- a/chrome/browser/importer/external_process_importer_client.h
 +++ b/chrome/browser/importer/external_process_importer_client.h
-@@ -24,10 +24,13 @@
+@@ -24,6 +24,7 @@
  #include "components/favicon_base/favicon_usage_data.h"
  #include "components/history/core/browser/history_types.h"
  #include "mojo/public/cpp/bindings/binding.h"
 +#include "net/cookies/canonical_cookie.h"
```

Comment:
this can be in chromium_src overrides too.
and did you forgot to commit the chromium_src overrides file?